lib/docker_manager/upgrader.rb

```diff
@@ -124,6 +125,11 @@ def upgrade
     percent(90)
     log("Running post deploy migrations")
     run("bundle exec rake multisite:migrate")
+
+    if using_s3_assets
+      run("bundle exec rake s3:expire_missing_assets")
```

Comment:
you forgot the less memory flags here @davidtaylorhq

Reply:
Fixed in #152